src/utils/hints/namesHints.js

```diff
@@ -0,0 +1,26 @@
+export const namesHints = {
+  name: 'Unique identifier that may be configured to point to accounts, contracts, oracles, and NFTs.',
+  nameInAuction: 'Name that points to accounts, contracts, oracles, NFTs and more. Names shorter than 13 characters need to be auctioned.',
+  expires: 'Estimated date, time, and block height when the name will be detached from the account.',
+  buyer: 'Address of the account which paid for the Name.',
+  owner: 'Owner of the Name.',
+  pointsTo: 'Address that Name points to.',
+  highestBid: 'Account of the highest bidder and the price of the highest bid.',
+  bidCount: 'Amount of bids in the auction.',
+  ends: 'Estimated date, time, and block height when the auction ends.',
+  expired: 'Estimated date, time, and block height when the Name was detached from the address.',
+  lastBuyer: 'Address of the account which paid for the Name.',
+  lastOwner: 'Last owner of the Name.',
+  bidder: 'Account of the highest bidder.',
+  bid: 'Price of the highest bid.',
+  ownedSince: 'Estimated date, time, and block height when the account became the owner of the Name.',
+  status: 'Current state of the Name in the blockchain.',
+  customPointers: 'User-defined pointers that can point to any valid value.',
+  accountPointer: 'Account that the Name points to.',
+  channelPointer: 'State Channel that the Name points to.',
+  contractPointer: 'Smart Contract that the Name points to.',
+  oraclePointer: 'Oracle that the Name points to.',
+  eventTransactionHash: 'Transaction hash that triggered the name event.',
```

Comment:
To me it makes the sense even simpler `of` instead of `that triggered`. Same for the lines above. As you wish, no strong preference.